examples/se_2_3_localization.cpp

```diff
@@ -46,26 +49,73 @@
+ *            (it is equivalent to p_dot in continuous time. This is usually called mixed-frame representation
+ *            and is denoted as (B[A] v_AB), where B[A] is the mixed frame as described above.
+ *            For reference, please see "Multibody Dynamics Notation" by Silvio Traversaro and Alessandro Saccon.
+ *            Link: https://research.tue.nl/en/publications/multibody-dynamics-notation-version-2)
+ * - a is the frame acceleration in mixed-representation (equivalent to p_doubledot in continuous time).
+ * - omega_b as the angular velocity of the robot expressed in the robot frame
+ *
+ * The kinematic equations (1) can be written as,
+ * p <-- p + v dt + 0.5 a dt^2
+ * R <-- R Exp_SO3(omega_b)
+ * v <-- v + a dt
+ *
+ * We would like to express these equations in the form,
+ * X <-- X* Exp(u)
+ * where, X \in SE_2(3), u \in R^9 and u_hat \in se_2(3)
+ * Note that here input vector u is expressed in the local frame (robot frame).
+ *
+ * The exponential mapping of SE_2(3) is defined as,
+ * for u = [u_p, u_w, u_v]
+ * Exp(u) = | Exp_SO3(u_w)   JlSO3(u_w) u_p   JlSO3(u_w) u_v |
+ *          | 0    0    0                 1                0 |
+ *          | 0    0    0                 0                1 |
+ * where, JlSO3 is the left Jacobian of the SO(3) group.
+ *
+ * Please see the Appendix C of the paper "A micro Lie theory for state estimation in robotics",
+ * for the definition of the left Jacobian of SO(3).
+ * Please see the Appendix D of the paper, for the definition of Exp map for SE(3).
+ * The Exp map of SE_2(3) is a simple extension from the Exp map of SE(3).
+ * Also, please refer to Example 7 of the paper to understand when and how the left Jacobian of SO(3)
+ * appears in the definitions of Exp maps. The Example 7 illustrates the scenario for SE(3).
+ * We use a direct extension here for SE_2(3).
+ * One can arrive to such a definition by following the convergent Taylor's series expansion
+ * for the matrix exponential of the Lie algebra element (Equation 16 of the paper).
+ *
+ * As a result of X <-- X * Exp(u), we get (2)
+ * p <-- p + R JlSO3(u_w) u_p
+ * R <-- R Exp_SO3(u_w)
+ * v <-- v + R JlSO3(u_w) u_v
+ *
+ * We would like to draw a relationship between the sets of equations (1) and (2).
+ * Additionally, we would also like to make use of the IMU measurements as exogeneous inputs.
+ * Considering R.T as the transpose of R, the IMU measurements are modeled as,
+ *    - linear accelerometer measurements alpha = R.T (a - g) + w_acc
+ *    - gyroscope measurements omega = omega_b + w_omega
+ * Note that the IMU measurements are expressed in the IMU frame (coincides with the robot frame - assumption).
+ * The IMU measurements are corrupted by noise,
+ *    - w_omega is the additive white noise affecting the gyroscope measurements
+ *    - w_acc is the additive white noise affecting the linear accelerometer measurements
+ *
```

Comment:
Add a note stating that IMU biases are not considered

Reply:
Done in https://github.com/artivis/manif/pull/154/commits/ded121b75f0f401c518336cf0d31ad862be50275